src/requirements.txt

```diff
@@ -1 +1,2 @@
 aws-assume-role-lib==2.10.0
+boto3==1.26.74
```

Comment:
There's a tradeoff to this... The lambda runtime includes boto3. So, yes, including boto3 as a packaging requirement means each release will package a specific version of boto3, presumably tested to work. Which is great. On the downside, the boto3 requirement also pulls in botocore, which significantly inflates the lambda package size. Not a big deal in general for the lambdas we're working with. But something to be aware of overall.